package.json

```
@@ -10,7 +10,7 @@
   "repository": "jshttp/http-assert",
   "dependencies": {
     "deep-equal": "~1.0.1",
-    "http-errors": "~1.8.0"
+    "http-errors": "^2.0.0"
```

```suggestion
    "http-errors": "~2.0.1"
```

We support Node@0.8 so `^` is not an option, also there is the version http-errors@2.0.1 available